src/LiveDevelopment/LiveDevMultiBrowser.js

```diff
@@ -719,6 +719,10 @@ define(function (require, exports, module) {
     /**
      * For files that don't support as-you-type live editing, but are loaded by live HTML documents
      * (e.g. JS files), we want to reload the full document when they're saved.
+     * CSS files do support as-you-type live editing, however, once they are saved, the live
```

Comment:
Add `Bramble note:` to the start of this sentence, so it's clear it's us doing it.